pinot-tools/src/main/java/org/apache/pinot/tools/RealtimeQuickStartWithMinion.java

```diff
@@ -63,7 +64,7 @@ public static void main(String[] args)
   }
 
   public Map<String, Object> getConfigOverrides() {
-    Map<String, Object> properties = super.getConfigOverrides();
+    Map<String, Object> properties = new HashMap<>();
```

Comment:
Suggest fixing the super class to create an empty hash map instead. We want to keep the configs from the base class